efficientdet/keras/inspector.py

```diff
@@ -135,6 +135,8 @@ def main(_):
       # use synthetic data if no image is provided.
       image_arrays = tf.ones((batch_size, *model_config.image_size, 3),
                              dtype=tf.uint8)
+      if FLAGS.only_network:
+        image_arrays = tf.cast(image_arrays, tf.float32)
```

Comment:
```image_arrays``` should between [0,1].

Reply:
Hi @fsx950223, thank you for your comment.
1. `Policy` object doesn't have `dtype` attribute, it only has `name`, `variable_dtype` and `compute_dtype`.
And is it really necessary to use global policy when the `ServingDriver` _always_ uses `float32` dtype when `only_network` is enabled? ([inference.py:313](https://github.com/google/automl/blob/f036f3cee588fee9ad93d01e8dc3db901eb95a85/efficientdet/keras/inference.py#L313))
2. `image_arrays` are in range [0, 255] only if `input_image` is provided. And unless the image exactly matches dimensions of the input tensor, benchmarking with `only_network` is not possible. The image would have to be resized. And the `resize` method always returns `float32` dtype - it solves both dimensions and dtype issues.

How about this?
```python
    if FLAGS.only_network:
      image_arrays = tf.image.resize(image_arrays, [*model_config.image_size])
    driver.benchmark(image_arrays, FLAGS.bm_runs, FLAGS.trace_filename)
```

Reply:
``` python
    if FLAGS.only_network:
      image_arrays = tf.image.convert_image_dtype(image_arrays,
                                                  tf.keras.mixed_precision.global_policy().compute_dtype)
      image_arrays = tf.image.resize(image_arrays, [*model_config.image_size])
    driver.benchmark(image_arrays, FLAGS.bm_runs, FLAGS.trace_filename)
```

Reply:
1. Even if `convert_image_dtype` yielded different dtype than `float32`, then `resize` method would immediately convert it to `float32` anyway. Is this step necessary?
2. Image size is already parsed on line 80, why parsing it again?

Reply:
Good, remove https://github.com/google/automl/blob/9bff7529b26efd5ef51b574995bdc60530c85ed5/efficientdet/keras/inspector.py#L106 too, and 
```
    if FLAGS.only_network:
      image_arrays = tf.image.convert_image_dtype(image_arrays, tf.float32)
      image_arrays = tf.image.resize(image_arrays, utils.parse_image_size(model_config.image_size))
    driver.benchmark(image_arrays, FLAGS.bm_runs, FLAGS.trace_filename)
```

Reply:
Done, thank you for your suggestions.